review_practice/react-app/src/containers/SearchBar/ModalBody/PriceFilter.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useContext } from 'react';
+
+import { SearchRoomContext } from 'pages/SearchRoom';
+
+import PriceRangeSlider from 'components/PriceRangeSlider';
+
+const MIN_PRICE = 0;
+const MAX_PRICE = 100000;
+
+export default ({ buttonState, setButtonState }) => {
+    // Context
+    const {
+        searchOption: {
+            searchOptions: { price },
+        },
+        dispatchSearchOption,
+    } = useContext(SearchRoomContext);
+
+    // State
+    const priceState = useState([
+        price && price.startPrice ? price.startPrice : MIN_PRICE,
+        price && price.endPrice ? price.endPrice : MAX_PRICE,
+    ]);
```

Comment:
```suggestion
    const priceState = useState([
        Math.min(price?.startPrice || 0, MIN_PRICE),
        Math.max(price?.endPrice  || 0, MAX_PRICE),
    ]);
```
값의 크기 비교는 `Math API`를 사용하면 더 직관적으로 표현할 수 있습니다!